facefusion/typing.py

```diff
@@ -6,6 +6,10 @@
 Kps = numpy.ndarray[Any, Any]
 Score = float
 Embedding = numpy.ndarray[Any, Any]
+Landmark = TypedDict('Landmark',
```

Comment:
FaceLandmarkSet = TypedDict('FaceLandmarkSet',
{   
	'5' : FaceLandmark68
	'68' : FaceLandmark68
})

Reply:
you mean `'5' : FaceLandmark5`?

Reply:
yes